tech-committee-charter.md

```diff
@@ -18,7 +18,7 @@ TC membership is developed through leadership in the project and was bootstrappe
 Changes to TC membership should be posted in the agenda document, and may be suggested as any other agenda item.
 
 ### No Over-Representation
-
+No more than one-fourth (25%) of the TC members may be affiliated with the same employer (in the event of confusion or concern, the OpenFeature TC will defer to the CNCF definition of "same employer"). If removal or resignation of a TC member, or a change of employment by a TC member, creates a situation where more than one-fourth of the TC membership shares an employer, then the situation must be immediately remedied by the resignation or removal of one or more TC members affiliated with the over-represented employer(s).
```

Comment:
Do you have a link to that definition?

Reply:
I wasn't able to find a formal definition on the CNCF site. This is something we should address before merging. If the CNCF no longer has a published definition, we should either ask them to republish it or defined it ourselves.

I also noticed this was the same language used in the [OTel TC charter](https://github.com/open-telemetry/community/blob/main/tech-committee-charter.md).

Reply:
Same employer might be misleading in some cases (e.g. contract work which is pretty common in OSS). I would rephrase to "representing the same legal entity/project"

Reply:
OK, this is about the TC. As the TC has no formal "power", I am not sure how strict we need to be here.

Reply:
I think it's not a bad idea anyway.